spacy/lang/ru/lemmatizer.py

```diff
@@ -110,13 +119,19 @@ def pymorphy2_lemmatize(self, token: Token) -> List[str]:
     def pymorphy2_lookup_lemmatize(self, token: Token) -> List[str]:
```

Comment:
I find the naming here confusing. Might it make sense to rename these methods to `pymorphy_(lookup_)lemmatize()` and if necessary for backwards compatibility to retain `pymorphy2`- and `pymorphy3`-specific methods that call them?

Reply:
In the `Lemmatizer`, the specific lemmatizer method derived from the mode name, so if the mode is `pymorphy2_lookup`, then `__call__` calls `pymorphy2_lookup_lemmatize`, so this is intentional and correct (for the current clunky design, at least).

Reply:
I feel it would be clearer to have general methods `pymorphy_lookup_lemmatize()` and `pymorphy_lemmatize()` that can be wrapped by empty shell methods `pymorphy2...` and `pymorphy3...` At the moment in `lemmatizer.py` we call the `pymorphy2...` method immediately after the decision block that has established we may in fact be using pymorphy3, which looks like a mistake.

Reply:
`pymorphy_lemmatize` could be confusing as a name because if you accidentally used `pymorphy` as the mode you'd get weird behavior, but I think we can use private methods instead to make this clearer.